src/components/graph-view.js

```diff
@@ -1122,6 +1124,7 @@ class GraphView extends React.Component<IGraphViewProps, IGraphViewState> {
         nodeKey={nodeKey}
         viewWrapperElem={this.viewWrapper.current}
         isSelected={this.isEdgeSelected(edge)}
+        edgeType={edge.edgeType}
```

Comment:
This doesn't seem to have anything to do with the diff. The Edge component can extract this information from the data property, if necessary. Please remove this from this PR and create a different one if this fixes a bug.